docs/topics_5/web-adding-custom-labels.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='web_adding_custom_labels_{context}']
+= Adding custom labels
+
+You can add custom labels to use in each project. When you add a label, it is disabled by default. If you no longer need a label, you can delete it.
+
+.Procedure
+
+. Click *Add label*.
+.. In the *Upload* tab, click *Browse* and select a labels file (`\*.mta.label.xml` or `*.windup.label.xml`).
+.. In the *Server path* tab, enter the *Labels path* of the server-side file or directory containing the label(s) in the text box, select *Scan recursively* if relevant, and click *Save*.
```

Comment:
Do not use (s). That screws up localization. It's OK to use the plural here. Also, they're not labels. They're label files or labels files. Although I'm not sure why you like "labels files". That sounds weird to me. I think they are label .XML files.